src/QuickMarcEditor/QuickMarcEditorRows/ControlNumberField/ControlNumberField.js

```diff
@@ -41,9 +88,13 @@ const ControlNumberField = ({
         parse={v => v}
       />
       {canSelectSourceFile && (
-        <SourceFileLookup
-          onSourceFileSelect={handleSourceFileSelection}
-        />
+        isLoadingHrid
+          ? <Loading data-testid="hridLoading" />

```

Comment:
Can we show loading next to the Authority file lookup button or make it disabled? Right now when hrid is loading the `SourceFileLookup` component is unmounted and it loses internal state, so selected authority file resets when open it again

Reply:
I've added both loading and disabled that button, since occasionally it is tricky to understand why there is no value with intermittent internet connection.
https://github.com/folio-org/ui-quick-marc/assets/77053927/6117e19f-4b41-4868-aa70-07d93a8c6b24